dist/environments

```diff
@@ -1 +1 @@
-KOBman
+KOBman,Aries-dev
```

Comment:
@asa1997  is this file required in every environment repo?

Reply:
Yes, for now. This file is used to move  the environment files from stage/ folder to envs/ folder during the installation of the utility